src/js/components/introduction.jsx

```diff
@@ -26,35 +24,31 @@ export class IntroductionComponent extends Component {
 
     constructor(...args) {
         super(...args);
-        this._debounceHeightCalculation = debounce(this.calculateIntroHeight.bind(this), 150);
     }
 
     componentDidMount() {
-        setTimeout(() => this.calculateIntroHeight());
-        window.addEventListener('resize', this._debounceHeightCalculation);
-    }
-
-    componentWillUpdate() {
-        this._debounceHeightCalculation();
+        // Height of Introduction component will be computed on render and on resize only
+        // Make sure Introduction component has fully rendered before computing height
+        setTimeout(() => {
+            this.calculateIntroHeight();
+        }, 200);
+
+        window.addEventListener('resize', () => {
```

Comment:
this should be debounced on resize otherwise it will be triggered and executed a lot of times during the resize.